Buildings/Fluid/HeatExchangers/AirToAirHeatRecovery/BaseClasses/Validation/Effectiveness.mo

```diff
@@ -68,22 +78,25 @@ Buildings.Fluid.HeatExchangers.AirToAirHeatRecovery.BaseClasses.Effectiveness</a
 The input signals are configured as follows:
 </p>
 <ul>
-<li>
-The temperature of the supply air, <i>TSup</i>, is larger than the temperature
-of the exhaust air, <i>TExh</i> before <i>20s</i>;
-After that, <i>TSup</i> is less than <i>TExh</i>, leading to a heating mode.
+<li>The temperature of the supply air, <i>TSup</i>, is larger than the temperature 
+of the exhaust air, <i>TExh</i>, before <i>20s</i>;
+After that, <i>TSup</i> is less than <i>TExh</i>, leading to a heating mode;
 </li>
+</ul>
+<ul>
 <li>
-The supply air flow rate, <i>vSup</i>, and the exhaust air flow rate,
-<i>vExh</i>, change from 0.3 to 1 and 0.2 to 1 
+The supply air flow rate, <i>VSup</i>, and the exhaust air flow rate, <i>VExh</i>, change from 
+0.6 to 1 and 0.8 to 1 
 during the period from <i>0s</i> to <i>60s</i>, respectively;
 They then stay constant.
 </li>
-<li>
-The wheel speed ratio, <i>y</i> keeps constant during the period from
+</ul>
+<ul>
+<li> 
+The wheel speed ratio, <i>wheSpe</i>, keeps constant during the period from 
 <i>0s</i> to <i>60s</i> and then increases from 0.3 to 1
 during the period from <i>60s</i> to <i>120s</i>.
-</li>
+</li> 
 </ul>
```

Comment:
How should the outputs look like? It may be more helpful to we can add some description of the outputs.